.github/check_agda.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -e
+
+export Agda_datadir=$PWD/data
+export PATH=$PWD/bin:$PATH
+
+if [[ "$OSTYPE" == "darwin"* ]]; then
+  # See https://gitlab.haskell.org/ghc/ghc/-/issues/17418
+  xattr -rc $PWD/bin
+fi
+
+echo "The nightly version of Agda is"
+agda --version
+
+echo "Checking if Agda nightly is runnable on your system..."
+for prim in ./data/lib/prim/Agda/**/*.agda
+do
+    echo "Checking "$prim
+    agda -v 2 $prim
+done
+
+if [ $? == 0 ];
```

Comment:
Unimportant nit: `==` is used for string comparisons. The result will be the same, but this is probably meant as a numeric comparison:
```diff
- if  [ $? == 0 ];
- then
+ if [ $? -eq 0 ]; then
```

Reply:
Humm... script languages should also be typed.